packages/embed/src/utils/create-iframe/handle-form-redirect.ts

```diff
@@ -0,0 +1,36 @@
+const updateUrlWithHash = (currentUrl: string, newUrl: string): string => {
+  if (!newUrl.includes('#')) {
+    return newUrl
+  }
+  if (!newUrl.startsWith(`${currentUrl}#`)) {
+    return newUrl
+  }
+  if (newUrl.includes('?')) {
+    return newUrl.replace('#', `&tf-embed-ts=${Date.now()}#`)
+  }
+  return newUrl.replace('#', `?tf-embed-ts=${Date.now()}#`)
+}
+export const handleFormRedirect =
+  (iframe: HTMLIFrameElement) =>
+  ({ url, target = '_parent' }: { url: string; target?: string }) => {
+    if (!url) {
+      // eslint-disable-next-line no-console
+      console.error('Redirect failed, no URL provided')
+      return
+    }
+
```

Comment:
wouldn't you prefer a switch in this case?

Reply:
Yes 🚀